planning/behavior_velocity_planner/src/scene_module/occlusion_spot/scene_occlusion_spot_in_private_road.cpp

```diff
@@ -82,18 +85,20 @@ bool OcclusionSpotInPrivateModule::modifyPathVelocity(
   nav_msgs::msg::OccupancyGrid occ_grid = *occ_grid_ptr;
   grid_map::GridMap grid_map;
   grid_utils::denoiseOccupancyGridCV(occ_grid, grid_map, param_.grid);
-  if (param_.show_debug_grid) {
-    publisher_->publish(occ_grid);
-  }
   double offset_from_start_to_ego = utils::offsetFromStartToEgo(interp_path, ego_pose, closest_idx);
+  using Slice = occlusion_spot_utils::Slice;
+  std::vector<Slice> detection_area_polygons;
+  utils::buildDetectionAreaPolygon(
+    detection_area_polygons, interp_path, offset_from_start_to_ego, param_);
   RCLCPP_DEBUG_STREAM_THROTTLE(logger_, *clock_, 3000, "closest_idx : " << closest_idx);
   RCLCPP_DEBUG_STREAM_THROTTLE(
     logger_, *clock_, 3000, "offset_from_start_to_ego : " << offset_from_start_to_ego);
   std::vector<utils::PossibleCollisionInfo> possible_collisions;
   // Note: Don't consider offset from path start to ego here
-  utils::generateSidewalkPossibleCollisions(
-    possible_collisions, grid_map, interp_path, offset_from_start_to_ego, param_,
-    debug_data_.sidewalks);
+  utils::generateDetectionAreaPossibleCollisions(
```

Comment:
getPossibleCollisionsinDetectionArea?

Reply:
I changed to createPossibleCollisionInDetectionArea
[b067661](https://github.com/autowarefoundation/autoware.universe/pull/354/commits/b0676615e86c71d3083f35e6221c0be97147a3a9)